lib/globals.js

```diff
@@ -26,6 +20,7 @@ exports.getHTMLCode = path => {
     html = nunjucks.renderString(content)
   } catch (err) {
     if (err) {
+      console.log(err)
```

Comment:
`console.error` should be safe to use here.

Reply:
I've removed this completely as it was left over from debugging and don't necessarily wont to be logging errors in prod without impact assessment?

Reply:
Edit: reintroduced as console.error as discussed